samples/subgroup-scan.cpp

```diff
@@ -0,0 +1,208 @@
+/***************************************************************************
+ *
+ *  Copyright (C) 2017 Codeplay Software Limited
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *  Codeplay's ComputeCpp SDK
+ *
+ *  scan-subgroups.cpp
+ *
+ *  Description:
+ *    Example of a parallel inclusive scan in SYCL using subgroup operations.
+ *
+ **************************************************************************/
+
+#include <CL/sycl.hpp>
+namespace sycl = cl::sycl;
```

Comment:
```suggestion
#include <sycl/sycl.hpp>
```
The `namespace sycl` bit isn't needed in 2020.

Reply:
I'm a bit confused - it looks as if you're not compiling/writing this as SYCL-2020, but sub-groups is a SYCL-2020 feature, and not a 1.2.1 feature?